app/alerts.py

```diff
@@ -99,18 +111,19 @@ def build_sites_markers(dpt_code=None):
 
     # We build a list of markers containing the info of each site/camera
     markers = []
-    for i, row in camera_positions.iterrows():
-        lat = row['Latitude']
-        lon = row['Longitude']
-        site_name = row['Tours']
-        nb_device = row['Nombres Devices']
-        markers.append(dl.Marker(id=f'site_{i}',    # Necessary to set an id for each marker to receive callbacks
+    for row in camera_positions:
+        id = row['id']
```

Comment:
```suggestion
        site_id = row['id']
```

It is not a big issue but [Codacy](https://app.codacy.com/gh/pyronear/pyro-platform/pullRequest?prid=6988642) has raised a warning due to this line because we are defining a variable under the name `id`, which is a built-in expression in Python, like `int`, `str`, `max`, `len`, etc. It does not break anything but it would be cleaner to use another name for the variable, like `site_id` for instance!

Reply:
I changed the name of the variable as you suggested it for site_id.
Thanks a lot @pechouc for the comments and the explanations !